### ISSUE TITLE
Using Stencil with other non-Stencil-related JSX components

### ISSUE THREAD

**Issue by tkbeal** [ionic-team/stencil#3285]
I am trying to build run `stencil build` but getting Typescript error for "Missing h import for JSX types..." in a non-Stencil-related folder of separately exported React components.

I am basically trying to export some custom web components in parallel with React components in the same npm-library. I have the `srcDir` in Stencil config pointed to the sub folder that only contains Stencil components, but the build is giving me errors for parents folder.

**Comment by rwaskiewicz** [comment]
Hey @tkbeal 👋 

I've transferred this issue over to the main Stencil repository rather than the documentation site.  Can you provide us with a minimal reproduction case (in the form of a GitHub repo) please? That would give the team something they could easily pull down and look into further.

**Comment by rwaskiewicz** [comment]
This issue is being closed due to the lack of a code reproduction. If this is still an issue with the latest version of Stencil, please create a new issue and ensure the template is fully filled out.

Thank you for using Stencil!
